lib/Horde/Text/Diff/Engine/Native.php

```diff
@@ -28,6 +28,51 @@
  */
 class Horde_Text_Diff_Engine_Native
 {
+    /**
```

Comment:
Interesting design issue: We have all these variables in the base class as "protected" but the only class that really inherits them is the "Native" engine.

Reply:
Yes, that was interested, but PHP 8.2 spammed me during testing and unit tests...